src/solver/conv_winoRxS_f3x2.cpp

```diff
@@ -218,19 +105,13 @@ bool ConvBinWinogradRxSf3x2::IsApplicable(const ConvolutionContext& params) cons
         return false;
     if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F3X2{}))
         return false;
-    if(!(params.direction.IsForward() || params.direction.IsBackwardData()))
-        return false;
     if(!params.use_asm_kernels)
         return false;
-    if(!params.rmv.IsV2orV3())
-        return false;
-    if(!params.IsLayoutDefault())
-    {
+    if(!params.rmv.IsV3())
```

Comment:
Do we support non-NCHW layout now?

Reply:
No, we dont. Constraint checked in the `IsShaderContraintsMet`: https://github.com/ROCmSoftwarePlatform/MIOpen/blob/bcf87b840cb07f728582bf745b9f76bf9565a220/src/solver/conv_winoRxS_f3x2.cpp#L71-L74

However, this check is not obvious, so it was fixed in e105526

Reply:
I see, thanks.